firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -215,6 +233,23 @@ private void ensureClientConfigured() {
     }
   }
 
+  private FirebaseFirestoreSettings mergeEmulatorSettings(
```

Comment:
Uh oh, now all my code leaked in here :-/

Reply:
Oops, will fix!

Reply:
@samtstern for some reason I can't revert that commit locally. I'll close this PR and open a clean one

Reply:
@rosariopfernandes you can also just force-push onto this same branch, GitHub will handle it.

Reply:
@samtstern done! Thanks for the tip